### ISSUE TITLE
Tailwind shoud be the default style

### ISSUE THREAD

**Issue by beeman** [nxpm/stack#56]
## Expected Behavior

Tailwind is the default style.

## Current Behavior

Bootstrap is the default style.

## Possible Solution

Search for `bootstrap` and `tailwind` and make sure to switch any place where `bootstrap` takes precedence.

## Requirements

- dark mode should be configured using 'class'
- `<body class="dark">` should be set in index.html
- the @tailwindcss/forms plugin should be installed.
- use 'css' instead of 'scss'

**Comment by ruslanguns** [comment]
+1

**Comment by ruslanguns** [comment]
Will you keep scss as default with Tailwind?

**Comment by beeman** [comment]
> Will you keep scss as default with Tailwind?

I think it's better to use css. Should make compilation a bit faster as it's processed once. 

Actually I already tried to change this to work around an issue I had with tailwind's dark mode (which was unrelated, fixed right now), but I [reverted the change](https://github.com/nxpm/stack/commit/a2870d982c6f2eb1933f1397d11dc71535643df0).

I added it to the requirements.
